src/main/java/net/helpscout/api/model/thread/BaseLineItem.java

```diff
@@ -1,99 +1,49 @@
 package net.helpscout.api.model.thread;
 
+import java.util.Date;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.experimental.FieldDefaults;
 import net.helpscout.api.cbo.ActionType;
 import net.helpscout.api.cbo.Status;
 import net.helpscout.api.model.ref.MailboxRef;
 import net.helpscout.api.model.ref.PersonRef;
 import net.helpscout.api.model.ref.UserRef;
 
-import java.util.Date;
-
+@Data
+@FieldDefaults(level = AccessLevel.PRIVATE)
 public class BaseLineItem implements LineItem {
-	private UserRef assignedTo;
-	private Status status;
-	private PersonRef createdBy;
-	private Date createdAt;
-	private MailboxRef fromMailbox;
-    private ActionType actionType;
-    private Long actionSourceId;
+    
+    UserRef assignedTo;
+    Status status;
+    PersonRef createdBy;
+    Date createdAt;
+    MailboxRef fromMailbox;
+    ActionType actionType;
+    Long actionSourceId;
 
 	public boolean isAssigned() {
-		return this.assignedTo != null;
+	    return getAssignedTo() != null;
 	}
 
 	public boolean isActive() {
-		return this.status == Status.Active;
+	    return getStatus() == Status.Active;
```

Comment:
Yeah, I 100% agree, but unfortunately we need to keep these around (until v 2.0) to ensure backward compatibility.